packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -239,7 +254,8 @@ export class ClusterReachability {
         }
 
         if (e.candidate.type === CANDIDATE_TYPES.RELAY) {
-          this.storeLatencyResult('tcp', this.getElapsedTime());
+          const protocol = e.candidate.port === TURN_TLS_PORT ? 'tls' : 'tcp';
```

Comment:
this code here relies on the assumption that we will always get TLS URLs with port 443 from Orpheus, I guess it's fine, but maybe it would be better for us to make sure that it is always 443 and set the port to 443 inside `convertStunUrlToTurnTls()`?

Reply:
Done